virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,71 +1,40 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Attaching a virtual machine to an external network
```

Comment:
Petr suggested calling this a Linux bridge network. What do you think?

Reply:
We just changed this to _Configuring a virtual machine network interface to use the default network_

Reply:
Changed the title of this assembly to: _Configuring a virtual machine network interface to use a Linux bridge network_